files/grest/rpc/stake_address/stake_distribution.sql

```diff
@@ -0,0 +1,156 @@
+CREATE TABLE IF NOT EXISTS GREST.STAKE_DISTRIBUTION (
+    STAKE_ADDRESS varchar PRIMARY KEY,
+    POOL_ID varchar,
+    TOTAL_BALANCE numeric,
+    UTXO numeric,
+    REWARDS numeric,
+    WITHDRAWALS numeric,
+    REWARDS_AVAILABLE numeric,
+    RESERVES numeric,
+    TREASURY numeric
+);
+
+CREATE TABLE IF NOT EXISTS GREST.CONTROL_TABLE (
+    control_type text PRIMARY KEY,
+    value text NOT NULL
+);
+
+DO $$
+DECLARE
+    _last_accounted_block_height bigint;
+BEGIN
+    SELECT
```

Comment:
Might be unnoticeable (ms) but we could use MAX function here to pluck biggest value from index